### ISSUE TITLE
make php 7 work

### ISSUE THREAD

**Issue by ghost** [OfficeStack/OpenSkedge#114]
I want to user OpenSkedge on Debian 9 with PHP 7. This leads to the following problem:
```
PHP Warning:  Erroneous data format for unserializing 'OpenSkedge\\AppBundle\\Entity\\User' in /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php on line 833, referer: https://…/login/
PHP Notice:  unserialize(): Error at offset 43 of 44 bytes in /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php on line 833, referer: https://…/login/
PHP Fatal error:  Uncaught Error: __clone method called on non-object in /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php:837\nStack trace:\n#0 /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php(2393): Doctrine\\ORM\\Mapping\\ClassMetadataInfo->newInstance()\n#1 /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php(2474): Doctrine\\ORM\\UnitOfWork->newInstance(Object(Doctrine\\ORM\\Mapping\\ClassMetadata))\n#2 /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php(246): Doctrine\\ORM\\UnitOfWork->createEntity('OpenSkedge\\\\AppB...', Array, Array)\n#3 /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php(479): Doctrine\\ORM\\Internal\\Hydration\\ObjectHydrator->_getEntity(Array, 'r')\n#4 /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php(150): Doctrine\\ORM\\Internal\\Hydration\\ObjectHydrator->hydrateRowData(Array, Array, Array)\n#5 /var/www/openskedge/v in /var/www/openskedge/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php on line 837, referer: https://…/login/
```
I would also offer to test everything that is necessary to port on php7.

**Comment by maxfierke** [comment]
@YoSiJo folks have had success running it on PHP 7.x with the `1.2.x-dev` branch: (e.g. https://github.com/OfficeStack/OpenSkedge/issues/113). IIRC, I updated the Doctrine version on that branch to one that's compatible with PHP 7.

**Comment by ghost** [comment]
Good day,
I am sorry, but due to the takeover of github by Microsoft I will delete my account here.

I hope that the problem will be solved.

**Comment by Adwooo** [comment]
Hey thanks for your message.

Are you moving it to some other repository?

What are a bad site's of Microsoft taking over git hub so you divided to
quit?


Adriab

pon., 4 cze 2018, 22:18 użytkownik Max Fierke <notifications@github.com>
napisał:

**Comment by maxfierke** [comment]
Nope. Not moving it anywhere. I am fine with the Microsoft acquisition. I closed this issue because 1.2.x should work PHP 7.
